src/styles/containerComposeStyles/index.native.ts

```diff
@@ -1,6 +1,5 @@
-import styles from '@styles/styles';
 import ContainerComposeStyles from './types';
 
-const containerComposeStyles: ContainerComposeStyles = [styles.textInputComposeSpacing];
+const containerComposeStyles: ContainerComposeStyles = (stylesObject) => [stylesObject.textInputComposeSpacing];
```

Comment:
can we rename this to just `styles`? :)

Reply:
@lukemorawski would you be able to update with this change today?